src/main/java/teammates/common/util/FileHelper.java

```diff
@@ -27,9 +27,11 @@ public static InputStream getResourceAsStream(String file) {
     public static String readResourceFile(String file) {
         InputStream is = getResourceAsStream(file);
         Scanner scanner = new Scanner(is, Const.SystemParams.ENCODING);
-        String content = scanner.useDelimiter("\\Z").next();
-        scanner.close();
-        return content;
+        try {
+            return scanner.useDelimiter("\\Z").next();
+        } finally {
+            scanner.close();
+        }
```

Comment:
[Java's try-with-resource](https://meebleforp.com/blog/41/analysing-student-software-engineering-projects-metadata) may be applicable here. If it is, can be a firstTimer issue.